src/Customers/Entry/Property.php

```diff
@@ -49,11 +57,9 @@ public function hydrate(array $data, array $embedded = [])
     public function extract(): array
     {
         return [
-            'type' => $this->getType(),
-            'slug' => $this->getSlug(),
-            'name' => $this->getName(),
+            'op' => $this->getOperation(),
             'value' => $this->getValue(),
-            'source' => $this->getSource(),
```

Comment:
Thanks for taking the initiative on this PR!  Unfortunately this method needs to remain exporting all/most properties so that we can maintain consistency across all entities within the SDK.  Typically `extract` is also used to obtain all properties during Creation, so to reduce it to only `value` could be an issue down the road.